packages/zowe-explorer/src/extension.ts

```diff
@@ -363,7 +363,7 @@ async function watchConfigProfile(context: vscode.ExtensionContext) {
             new vscode.RelativePattern(vscode.workspace.workspaceFolders[0].uri.fsPath, "zowe.config.user.json")
         );
 
-    context.subscriptions.push(globalProfileWatcher);
+    context.subscriptions.push(workspaceProfileWatcher);
     workspaceProfileWatcher && context.subscriptions.push(globalProfileWatcher);
```

Comment:
@kristinochka I didn't see it when testing your PR but did notice while working on another branch.  This had a double subscription on the same watcher and would result in a stack error in the debug console when deleting a config that was being watched in workspace.  This changed fixed that issue.